docs/reference/synonyms/apis/synonyms-apis.asciidoc

```diff
@@ -0,0 +1,36 @@
+[[synonyms-apis]]
+== Synonyms APIs
+
+beta::[]
+
+++++
+<titleabbrev>Synonyms APIs</titleabbrev>
+++++
+
+---
+
+The synonyms management API provides a convenient way to define and manage synonyms in an internal system index. Related synonyms can be grouped in a "synonyms set".
+Create as many synonym sets as you need.
+
+This provides an alternative to:
+- Defining inline synonyms in an analyzer definition, which impacts mapping size and can lead to performance issues.
```

Comment:
```suggestion

- Defining inline synonyms in an analyzer definition, which impacts mapping size and can lead to performance issues.
```

Reply:
Without this newline the list doesn't render correctly.

Reply:
Good catch, thanks! 👍